news-bundle/src/Resources/contao/templates/news/news_latest.html5

```diff
@@ -1,22 +1,22 @@
 
-<div class="layout_latest arc_<?= $this->archive->id ?> block<?= $this->class ?>" itemscope itemtype="http://schema.org/Article">
+<div class="layout_latest arc_<?= $this->archive->id ?> block<?= $this->class ?>">
```

Comment:
schema.org data only makes sense in detail templates. There is no such thing as describing lists. This data has been irrelevant ever since. What matters is that the search engines know if a page is describing a product or a news article etc. - the list views themselves do not matter.
Also see https://developers.google.com/search/docs/data-types/product
> We recommend focusing on adding markup to product pages instead of pages that list products or a category of products.